src/python/metrics/fuzzer_stats.py

```diff
@@ -226,6 +226,10 @@ def __init__(self, fuzzer, job, build_revision, timestamp):
         'kind': 'TestcaseRun',
     })
 
+    source = environment.get_value('STATS_SOURCE')
+    if source:
+      self._stats_data['source'] = source
```

Comment:
Should there be a default ?

Reply:
I don't think we should have a default, since all the existing stats won't have it populated. In any case I think having this blank is equivalent to any arbitrary value we pick as the default.